rfd/0026/README.md

```diff
@@ -890,6 +890,20 @@ exception is that the `uuid` field is named `id` to adhere to current
 conventions between the representation of Triton objects in CloudAPI and
 internal APIs.
 
+#### New `volumes` parameter/property for CreateMachine
+
+When creating a machine via CloudAPI, the new `volumes` parameter will allow
+one to specify a list of volumes to mount in the new machine. This would look
+as follows in the CreateMachine payload:
+
+```
+"volumes": ["volume-name-1", "volume-name-2", ...]
+```
+
+and the new machine would then have the specified volumes mounted when it
+starts, and the appropriate references will be added to indicate that this
```

Comment:
Maybe use the term "volume references" instead of just "references" to make it clear that these references are on volume objects, not machine objects?